throttling/src/test/java/com/iluwatar/throttling/B2BServiceTest.java

```diff
@@ -36,14 +36,15 @@
   @Test
   public void dummyCustomerApiTest() {
     Tenant tenant = new Tenant("testTenant", 2);
-    Throttler timer = new ThrottleTimerImpl(10);
+    Throttler timer = new ThrottleTimerImpl(100);
     B2BService service = new B2BService(timer);
-    
+
+    long counter = 0;
```

Comment:
Can you explain briefly why are these changes needed?

Reply:
To be honest, I am unable to figure out this. The code works perfectly this way but if the counter is calculated outside the for loop(like done previously), then sometimes it is not reflecting the proper changes.

Reply:
I am sorry. I think I tested this method before incrementing the count limit from 10 to 100(which was the root of intermittent failure). Will change this back to the original